Bottom: 6da49fd13727cb2ce93cf5d46772b298a2a9d047
Top:    06fa4b1e7148e4b21ed0293ea5fbc76e73eedcb3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 14:09:29 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
